olive/passes/onnx/nvmo_quantization.py

```diff
@@ -40,60 +33,433 @@ class Precision(StrEnumBase):
         INT4 = "int4"
 
     class Algorithm(StrEnumBase):
-        RTN = "RTN"
         AWQ = "AWQ"
 
+    class Calibration(StrEnumBase):
+        AWQ_LITE = "awq_lite"
+        AWQ_CLIP = "awq_clip"
+
     @classmethod
     def _default_config(cls, accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
         return {
             "precision": PassConfigParam(
                 type_=NVModelOptQuantization.Precision,
                 default_value="int4",
-                search_defaults=Categorical(["fp8", "int8", "int4"]),
+                searchable_values=Categorical(["fp8", "int8", "int4"]),
                 description="NVModelOpt Quantization mode.",
             ),
             "algorithm": PassConfigParam(
                 type_=NVModelOptQuantization.Algorithm,
-                default_value="RTN",
-                search_defaults=Categorical(["RTN", "AWQ"]),
-                description="Algorithm of weight only quantization. Support 'RTN' and 'AWQ'.",
+                default_value="AWQ",
+                searchable_values=Categorical(["AWQ"]),
```

Comment:
Will this always be the default value? Any plan to add more algorithm in the future?

Reply:
Is `awq-lite` the default algorithm? @anujj

Reply:
Yes Riyad